src/content/interop/js-interop/package-web.md

```diff
@@ -156,6 +156,78 @@ Node append(Node node) native;
 `native` is an internal keyword that means the same as `external` in this
 context.
 
+### Element and HTMLElement
+
+`Element` is now `HTMLElement` but `Element` class also exists as base class for
+HTMLElement, SVGElement etc. See [mdn element].
+`querySelector` returns `Element` as it can query in SVG as well so cast is
+required to access `HTMLElement` methods.
+
+```dart
+element.querySelector('#selectme')!.className = 'test'; // valid in both
+
+element.querySelector('#selectme')!.style.color = 'red'; // Remove
+(element.querySelector('#selectme') as HTMLElement).style.color = 'red'; // Add
+```
+
+### List operations
+
+Methods like `Element::querySelectorAll` or `Element::children` return values no
+longer implement `List` interface.
+
+For immutable operations you can use `JSImmutableListWrapper`:
+
+```dart
+final anchors = document.querySelectorAll('a');
+for (final anchor in anchors) {} // Remove
+for (final anchor in JSImmutableListWrapper(anchors)) {} //Add
+
+for (final child in parent.children) {} // Remove
+for (final child in JSImmutableListWrapper(parent.children)) {} // Add
+```
+
+You need to add implementation for simple mutable operations like `removeWhere`:
+
+```dart
+parent.children.removeWhere(test); // Remove
+
+for (var i = parent.children.length - 1; i >= 0; --i) {
+  if (test(parent.children.item(i)!)) {
+    parent.children.item(i)!.remove();
+  }
+}  // Add
+
+```
+
+### Common DOM manipulation examples
+
+All non standard `dart:html` fields and helpers not present in IDL definitions
```

Comment:
nit: I think we can just reword this to something like:

Here are some common examples of trivial changes that need to be made when migrating from `dart:html` to `package:web`: